apps/examples/testcase/ta_tc/media/utc/utc_media_mediaplayer.cpp

```diff
@@ -48,13 +48,21 @@ void EmptyObserver::onPlaybackPaused(media::MediaPlayer &mediaPlayer)
 {
 }
 
-static void SetUp(void)
+static int SetUp(void)
 {
 	FILE *fp = fopen(dummyfilepath, "w");
-	fputs("dummydata", fp);
-	fclose(fp);
+	if (fp != nullptr) {
+		if (fputs("dummydata", fp) != EOF) {
```

Comment:
below is correct.
```
size_t ret = fputfs("dummydata", fp);
fclose(fp);
if (ret != strlen("dummydata") {
     return TRUE;
}
return false;
```

Reply:
int fputs(FAR const char *s, FAR FILE *stream)
{
	int nput;
	int ret;

	/* Make sure that a string was provided. */

#ifdef CONFIG_DEBUG				/* Most parameter checking is disabled if DEBUG is off */
	if (!s) {
		set_errno(EINVAL);
		return EOF;
	}
#endif

	/* Write the string.  Loop until the null terminator is encountered */

	for (nput = 0; *s; nput++, s++) {
		/* Write the next character to the stream buffer */

		ret = lib_fwrite(s, 1, stream);
		if (ret <= 0) {
			return EOF;
		}

		/* Flush the buffer if a newline was written to the buffer */

		if (*s == '\n') {
			ret = lib_fflush(stream, true);
			if (ret < 0) {
				return EOF;
			}
		}
	}

	return nput;
}

Reply:
EOF is error, but we should check size of written data.